azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1335,6 +1448,38 @@ func flattenKubernetesClusterNetworkProfile(profile *containerservice.NetworkPro
 		values["load_balancer_sku"] = string(profile.LoadBalancerSku)
 	}
 
+	lbProfiles := make([]interface{}, 0)
+
+	if profile.LoadBalancerProfile != nil {
+		lb := make(map[string]interface{})
+
+		if profile.LoadBalancerProfile.ManagedOutboundIPs != nil {
+			lb["managed_outbound_ips"] = profile.LoadBalancerProfile.ManagedOutboundIPs.Count
+		}
+
+		if profile.LoadBalancerProfile.OutboundIPs != nil {
+			lb["outbound_ips"] = profile.LoadBalancerProfile.OutboundIPs.PublicIPs
+		}
+
+		if profile.LoadBalancerProfile.OutboundIPPrefixes != nil {
+			lb["outbound_ip_prefixes"] = profile.LoadBalancerProfile.OutboundIPPrefixes.PublicIPPrefixes
+		}
+
+		if profile.LoadBalancerProfile.EffectiveOutboundIPs != nil {
+			effectiveIps := make([]string, 0)
+
+			for _, ip := range *profile.LoadBalancerProfile.EffectiveOutboundIPs {
+				effectiveIps = append(effectiveIps, *ip.ID)
```

Comment:
there's a potential crash here if `ip.ID` is nil - as such could we nil-check this?